var/spack/repos/builtin/packages/qmcpack/package.py

```diff
@@ -57,31 +59,69 @@ class Qmcpack(CMakePackage):
     # variant('+mixed', default=True, when='+cuda', description="...")
 
     # conflicts
-    conflicts('+soa', when='+cuda')
-    conflicts('^openblas+ilp64')
-    conflicts('^intel-mkl+ilp64')
+    conflicts(
+        '+phdf5',
+        when='~mpi',
+        msg='Parallel collective I/O requires MPI-enabled QMCPACK.\n \
+        Please add "~phdf5" to the Spack install line for serial QMCPACK.'
```

Comment:
This adds a bunch of space to the string, correct? See if you can do something like [this](https://stackoverflow.com/a/8578229/5828163).